govuk-docker.gemspec

```diff
@@ -0,0 +1,30 @@
+# -*- encoding: utf-8 -*-
+
+lib = File.expand_path("lib", __dir__)
+$:.unshift lib unless $:.include?(lib)
+
+require "govuk_docker/version"
+
+Gem::Specification.new do |s|
+  s.name         = "govuk-docker"
+  s.version      = GovukDocker::VERSION
```

Comment:
What about adding a licence here https://guides.rubygems.org/specification-reference/#license=

Reply:
Yeah, I thought about it, but I didn't want to put it in since I thought that might be better as an explicit PR which adds the licence. But perhaps we can just go ahead and give it the MIT licence?

Reply:
There's already a [licence](b8caed810a8414b40239047c0802b3e52c2ec3af) file in the repo, so don't think we need a explicit PR to add a licence here.